inviteme/main.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/gorilla/schema"
+	"net/http"
+	"net/url"
+)
+
+const (
+	inviteLink    = "https://jdsb-test-ground-slack-invite.herokuapp.com/"
+	imageLink     = "http://giphygifs.s3.amazonaws.com/media/5q7XU3MurT0pW/giphy.gif"
+	imageText     = "I came in like a wrecking ball"
+	imageAltText  = "Picture of a cat wrecking neatly stacked books"
+	linkMsg       = "Click on the link below to get an invite to *jdsb-wrecking-ball* workspace: "
+	myJdsbSlackID = "UGAA0BJ4R"
+)
+
+var (
+	decoder = schema.NewDecoder()
+)
+
+func handler(request events.APIGatewayProxyRequest) (*events.APIGatewayProxyResponse, error) {
+	parsedBody, err := url.ParseQuery(request.Body)
+	if err != nil {
+		return &events.APIGatewayProxyResponse{
+			StatusCode: http.StatusBadRequest,
```

Comment:
Bad request is a 400 error, which blames the user for doing something wrong. This is inaccurate in the context of a Slack app. If no body comes in, that means Slack didn't send the body over. It's the service that screwed up in this case. So, an internal error (http status code 500) is accurate.

However, because of the way Slack reports any status code aside from 200 OK e.g. 400_user_error (no error message), you may want to return 200, then put the error message you want in the body. This would be more friendly to users.